src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -372,6 +372,10 @@ private void EndPreviousCommandIfNeeded()
             {
                 _countersOperation.EndPreviousCommandIfNeeded();
             }
+            else if (_inProgressCommand == CommandType.AttachmentPUT)
+            {
+                throw new InvalidOperationException("Please dispose the previous AttachmentPUT operation before starting a new one");
```

Comment:
This isn't needed.
There will be only one upload of an attachment, like a document